tests/ga/extension_emulator.py

```diff
@@ -18,22 +18,21 @@
 import json
 import re
 import uuid
-import os
 import contextlib
 from enum import Enum
 import subprocess
 
-import azurelinuxagent.common.utils.fileutil as fileutil
 import azurelinuxagent.common.conf as conf
 from azurelinuxagent.common.future import ustr
+from azurelinuxagent.common.exception import ExtensionError
 from tests.tools import Mock, patch
 
 from tests.protocol.mocks import HttpRequestPredicates
 
-from azurelinuxagent.ga.exthandlers import ExtHandlerInstance, HandlerManifest
+from azurelinuxagent.ga.exthandlers import ExtHandlerInstance
 
 
-class ExtensionCommandNames(Enum):
+class ExtensionCommandName(Enum):
```

Comment:
Wait I think you misread my comment @kevinclark19a. My comment was to make the comment in Line 152 singular as compared to making the whole class name singular lol. The class name should be plural, and now I understand what exactly you meant in the comment in line 152. Let me add another comment there.